tests/test_orders_trades_table.py

```
@@ -0,0 +1,398 @@
+
+    // 4. Test tab switching and localStorage persistence
+    switchOtTab('positions');
+    if (localStorage.getItem('crypto-spread-ot-view') !== 'positions') throw new Error('localStorage failed to save positions');
+    switchOtTab('trades');
+    if (localStorage.getItem('crypto-spread-ot-view') !== 'trades') throw new Error('localStorage failed to save trades');
+
+    console.log('ALL_JS_TESTS_PASSED');
+    process.exit(0);
+    """
+
+    res = subprocess.run(["node"], input=test_harness, capture_output=True, text=True, encoding="utf-8", timeout=5)
```

Ruff (0.16.3)

[error] Starting a process with a partial executable path

(S607)